internal/tag/tag.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tag
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/acr-cli/acr"
+	"github.com/Azure/acr-cli/cmd/api"
+	"github.com/pkg/errors"
+)
+
+// ListTags will do the http requests and return the digest of all the tags in the selected repository.
+func ListTags(ctx context.Context, acrClient api.AcrCLIClientInterface, repoName string) ([]acr.TagAttributesBase, error) {
+
+	lastTag := ""
+	resultTags, err := acrClient.GetAcrTags(ctx, repoName, "", lastTag)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to list tags")
+	}
+
+	var tagList []acr.TagAttributesBase
+	tagList = append(tagList, *resultTags.TagsAttributes...)
+
+	// A for loop is used because the GetAcrTags method returns by default only 100 tags and their attributes.
+	for resultTags != nil && resultTags.TagsAttributes != nil {
+		tags := *resultTags.TagsAttributes
+
+		// Since the GetAcrTags supports pagination when supplied with the last digest that was returned the last tag name
+		// digest is saved, the tag array contains at least one element because if it was empty the API would return
+		// a nil pointer instead of a pointer to a length 0 array.
+		lastTag = *tags[len(tags)-1].Name
+		resultTags, err = acrClient.GetAcrTags(ctx, repoName, "", lastTag)
+		if err != nil {
+			return nil, err
+		}
+		if resultTags != nil && resultTags.TagsAttributes != nil {
+			tagList = append(tagList, *resultTags.TagsAttributes...)
+		}
+	}
+
+	return tagList, nil
+}
+
+// DeleteTags receives an array of tags digest and deletes them using the supplied acrClient.
+func DeleteTags(ctx context.Context, acrClient api.AcrCLIClientInterface, loginURL string, repoName string, args []string) error {
```

Comment:
Nit. What is args, it looks like it is a tag list? Let's make sure the param name is self explanatory

Reply:
Addressed this.